pkgdb/src/buildenv/realise.cc

```diff
@@ -100,9 +101,11 @@ const char * const ZSH_ACTIVATE_SCRIPT = R"(
 if [ -d "$FLOX_ENV/etc/profile.d" ]; then
   declare -a _prof_scripts;
   _prof_scripts=( $(
-    echo "$FLOX_ENV/etc/profile.d"/*.sh;
+    cd "$FLOX_ENV/etc/profile.d";
+    shopt -s nullglob;
```

Comment:
```suggestion
```

Oops ... that was a cut & paste mistake on my part - we don't want to introduce the use of `shopt` when it wasn't there originally.